environment.yml

```diff
@@ -79,5 +81,4 @@ dependencies:
   - xlrd  # pandas.read_excel, DataFrame.to_excel, pandas.ExcelWriter, pandas.ExcelFile
   - xlsxwriter  # pandas.read_excel, DataFrame.to_excel, pandas.ExcelWriter, pandas.ExcelFile
   - xlwt  # pandas.read_excel, DataFrame.to_excel, pandas.ExcelWriter, pandas.ExcelFile
-  - pip:
-    - pyreadstat  # pandas.read_spss
+  - pyreadstat  # pandas.read_spss
```

Comment:
Can you revert this change here please. After that, you should run `./scripts/generate_pip_deps_from_conda.py` so `requirements.txt` is synchronized with the changes here.

Reply:
Sure, I will rebase mu branch from master and that should be reverterd